### PR TITLE
CLI support for ntp status

### PR DESCRIPTION
**Pull request by paweljakubas**
# Issue Number

#1427 

# Overview

- [x] I have extended network CLI cmd
- [x] I have added the test for it in CLISpec
- [x] I have added integration test

# Comments

### CODE REVIEW

File: lib/cli/test/unit/Cardano/CLISpec.hs [hunk 1]
```diff
@@ -469,6 +470,16 @@ spec = do
             , "  EPOCH_NUMBER             epoch number parameter or 'latest'"
             ]
 
+        ["network", "clock", "--help"] `shouldShowUsage`
+            [ "Usage:  network clock [--port INT]"
+            , "  View NTP offset."
```
Comment (Anviking) [inline lib/cli/test/unit/Cardano/CLISpec.hs:475]: _Maybe_ the description could be more elaborate such that users that might not even know what NTP is can understand what this command is useful for.

But there's a neatness to this conciseness too.

**Review by Anviking — APPROVED** [review]
lgtm!

**Review by KtorZ — APPROVED** [review]
LGTM, though the integration tests won't work on Windows.

File: lib/core-integration/src/Test/Integration/Scenario/CLI/Network.hs [hunk 2]
```diff
@@ -86,6 +94,12 @@ spec = do
             let maxEpoch = show $ fromIntegral @Word31 @Int maxBound
             params <- getNetworkParamsViaCliExpectingFailure ctx maxEpoch
             params `shouldContain` (errMsg404NoEpochNo maxEpoch)
+
+    it "CLI_NETWORK - network clock" $ \ctx -> do
+        eventually "ntp status = available" $ do
```
Comment (KtorZ) [inline lib/core-integration/src/Test/Integration/Scenario/CLI/Network.hs:102]: We need to mark this one as "pendingOnWindows" (see Test.Utils.Windows)

Comment (paweljakubas) [reply lib/core-integration/src/Test/Integration/Scenario/CLI/Network.hs:102]: done in https://github.com/input-output-hk/cardano-wallet/pull/1428/commits/d17692c6d1b554f325831040151bd8bfe446bb83

**Pull request merged**